src/subcommands/deployments.ts

```diff
@@ -401,11 +473,246 @@ function renderOverview(
   }
 }
 
-function renderJson(
+function renderShowJson(
   build: Build,
   project: Project,
   organization: Organization,
   databases: Database[],
 ) {
   console.log(JSON.stringify({ build, project, organization, databases }));
 }
+
+async function renderListOverview(
+  api: API,
+  project: Project,
+  databases: Database[],
+  deployments: Build[],
+  paging: PagingInfo,
+) {
+  const ingressRoot = new URL(endpoint()).hostname.split(".").at(-2);
+  for (;;) {
+    const table = deployments.map((build) => {
+      const status = deploymentStatus(project, build);
+      const colorByStatus = (s: string) =>
+        status === "Failed"
+          ? red(stripColor(s))
+          : status === "Production"
+          ? green(s)
+          : status === "Pending"
+          ? yellow(s)
+          : s;
+      const database = deploymentDatabase(databases, build);
+      if (isReady(status) && !database) {
+        error(
+          `Unexpected error: deployment uses a database not in the list of databases of the project`,
+        );
+      }
+      const databaseEnv = database
+        ? deploymentDatabaseEnv(project, database)
+        : "n/a";
+      const relativeDate = stripColor(
+        deploymentRelativeDate(build).split(", ")[0].trim(),
+      );
+      const date = `${deploymentLocaleDate(build)} (${relativeDate})`;
+      const row = {
+        Deployment: colorByStatus(build.deploymentId),
+        Date: colorByStatus(date),
+        Status: colorByStatus(status),
+        Database: colorByStatus(databaseEnv),
+        Domain: colorByStatus(
+          !isReady(status)
+            ? "n/a"
+            : `https://${project.name}-${build.deploymentId}.${ingressRoot}.dev`,
+        ),
```

Comment:
Why constructing a domain by hand, instead of obtaining it from `build.deployment.domainMappings`?

Reply:
I want to show only the domain with the deployment id. As deployments can have multiple domain mappings, I would have to search the array for that specific domain, thus coupling with the domain structure anyway. I think it is cleaner just constructing the domain by hand.

Reply:
changed to `sld`

Reply:
Thanks!